orangecontrib/text/corpus.py

```diff
@@ -55,6 +56,8 @@ def __init__(self, X=None, Y=None, metas=None, domain=None, text_features=None):
         self.text_features = None    # list of text features for mining
         self._tokens = None
         self._dictionary = None
+        self._ngrams_dictionary = None
```

Comment:
This is open for discussion but I'm not sure if I like this. What are the benefits of having both dictionaries: `dictionary` and `ngrams_dictionary`? Was this added to support access to both tokens and ngrams dictionaries? If so, what are some use cases for this? What are the schemas where this would be necessary?

For me the winning argument for single access point is still that other widgets do not have to bother and hence can seamlessly work with either one. Consequently we would also need just one dictionary.